arch/arm/src/nrf52/Make.defs

```diff
@@ -89,7 +89,7 @@ ifeq ($(CONFIG_ARCH_CHIP_NRF52832),y)
 CHIP_CSRCS += nrf52832_errdata.c
 endif
 
-CHIP_CSRCS += nrf52_timerisr.c
+CHIP_CSRCS += nrf52_systimer.c
```

Comment:
need rename the comment "arch/arm/src/nrf52/nrf52_timerisr.c" to "arch/arm/src/nrf52/nrf52_systimer.c"

Reply:
OK